packages/jest-source-map/src/index.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+export {default as getCallsite} from './getCallsite';
```

Comment:
Just a thought, but maybe worth to only use named exports everywhere?

Reply:
I'm down with that. there's https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-default-export.md to enforce it